README.md

```diff
@@ -42,6 +42,16 @@ Just add the view within the `renderItem` slot in `listing/partials/item.blade.p
 @includeWhen(config('frontend.category'), 'AmastyPromoBanners::promobanners.amongproducts')
 ```
 
+You should also use `getListingSize(...)` as the page size in your `products.blade.php`, for example like so:
+
+```blade
+<reactive-list
+    [...]
+    :size="getListingSize(isNaN(parseInt(listingSlotProps.pageSize)) ? 10000 : parseInt(listingSlotProps.pageSize))"
```

Comment:
What is the 10000 here? Seems like a massive amount of products which could cause your page to crash and cause large server load.

Reply:
This comes from standard rapidez. We can definitely change it there though, I also don't see the use for it.